### PR TITLE
Added first week closed on registration.php

### PR DESCRIPTION
**Pull request by Aholicknight**
I noticed the first week is closed and it will help out people. Let's say they enter all the registration and find out the first week is closed at the end.

### CODE REVIEW

File: www/index.php [hunk 1]
```diff
@@ -47,6 +47,7 @@
       <a class="twitter" href="https://twitter.com/compcamps"><i class="fab fa-3x fa-twitter"></i></a>
       <a class="steam" href="http://steamcommunity.com/groups/msicompcamps"><i class="fab fa-3x fa-steam"></i></a>
       <a class="facebook" href="https://www.facebook.com/compcamps"><i class="fab fa-3x fa-facebook"></i></a>
+      <a class="reddit" href="https://www.reddit.com/r/compcamps/"><i class="fab fa-3x fa-reddit></i></a>
```
Comment (BrettMayson) [inline www/index.php]: This is unrelated to this pull request and needs to be removed

File: www/register.php [hunk 2]
```diff
@@ -9,6 +9,8 @@
 <section class="register-section">
   <h2>Registration is Open</h2>
   <h3>$350</h3>
+  <h3>First week: <b>CLOSED</b></h3>
+  <h3>Secound week: OPEN</h3>
```
Comment (BrettMayson) [inline www/register.php]: This is not spelled correctly

**Review by BrettMayson — CHANGES_REQUESTED** [review]

File: www/register.php (hunk 2 above)
Comment (trollmad3) [reply www/register.php]: Isn't it spelt week?

Comment (Aholicknight) [reply www/register.php]: Typo fixed.

**Review by Aholicknight — COMMENTED** [review]
Fixed typo.

**Review by BrettMayson — APPROVED** [review]

**Pull request merged**